official-antistasi-community/A3-Antistasi
# Meer MIE phase 5 (FFAA)

## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
    Expands and reintegrates FFAA as a full template. Major change to selector.sqf to allow for this.

### Please specify which Issue this PR Resolves.
closes none

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in singleplayer?
2. [x] Have you loaded the mission in LAN host?
3. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
  Load one of each environment with FFAA loaded and check that the units are spawning.
********************************************************
Notes:
Only Temp and Arid available, it's all they have.
